ui-lib/scripts/buildReactIcons.js

```diff
@@ -0,0 +1,119 @@
+const fs = require("fs");
+const glob = require("glob");
+const camelcase = require("camelcase");
+const path = require("path");
+const svgr = require("@svgr/core").default;
+
+const rootDir = path.join(__dirname, "../src/assets/icons");
+const reactDir = `${rootDir}`;
+// const reactNativeDir = `${rootDir}/reactNative`;
```

Comment:
Can we just remove comments from this file, please?

Reply:
Sure, removed all traces of rn code